test/unit-tests/components/viewmodels/roomlist/RoomListViewModel-test.tsx

```diff
@@ -185,6 +187,33 @@ describe("RoomListViewModel", () => {
             expect(fn).toHaveBeenLastCalledWith(expect.arrayContaining([FilterKey.MentionsFilter]));
         });
 
+        it("should remove all filters when active space is changed", async () => {
+            mockAndCreateRooms();
+            const { result: vm } = renderHook(() => useRoomListViewModel());
+
+            // Let's first toggle the People filter
+            const i = vm.current.primaryFilters.findIndex((f) => f.name === "People");
+            act(() => {
```

Comment:
the docs say sync act is deprecated so I would be tempted to await on all of them for consistency?